### ISSUE TITLE
Key "0" does not exist as the array is empty.

### ISSUE THREAD

**Issue by echantigny** [verbb/feed-me#383]
### Description
Since the last update, I can't get into my feeds.  

### Steps to reproduce

1. Go in Feed Me
2. Click on a feed with dev mode enabled.

### Additional info

- Plugin version: 3.0.0-beta.21
- Craft version: 3.0.21
- Craft Commerce: 2.0.0-beta.8

Output of the error pointing clearly at line 31, of \vendor\verbb\feed-me\src\templates\_includes\elements\entry\groups.html

```Twig_Error_Runtime: Key "0" does not exist as the array is empty. in \vendor\verbb\feed-me\src\templates\_includes\elements\entry\groups.html:31
Stack trace:
#0 \vendor\craftcms\cms\src\helpers\Template.php(73): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig_Source), Array, 0, Array, 'array', false, false)
#1 \storage\runtime\compiled_templates\4b\4b3a9b6f858a70982005afd6951878fda855f9aa1eb82edfa5de9d7c228bc23b.php(95): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig_Source), Array, 0, Array, 'array')
#2 \vendor\twig\twig\lib\Twig\Template.php(390): __TwigTemplate_00ae3650c9b0773557621a3b933302d280fb6de771f811ab40ce165e3cd1d0c4->doDisplay(Array, Array)
#3 \vendor\craftcms\cms\src\web\twig\Template.php(49): Twig_Template->displayWithErrorHandling(Array, Array)
#4 \vendor\twig\twig\lib\Twig\Template.php(367): craft\web\twig\Template->displayWithErrorHandling(Array, Array)
#5 \vendor\craftcms\cms\src\web\twig\Template.php(31): Twig_Template->display(Array, Array)
#6 \storage\runtime\compiled_templates\da\da8e5f8430fa27f5d100316f9ff89bd64d231217175040d007619d512ef214b1.php(176): craft\web\twig\Template->display(Array)
#7 \vendor\twig\twig\lib\Twig\Template.php(189): __TwigTemplate_b6faf5617e2ccd3b7dfdbc08678649b2eab5a6cc5f94b98453356dccdc895739->block_blockContent(Array, Array)
#8 \storage\runtime\compiled_templates\cd\cd015d2ba4762dfa6ba2c7191b3f3684b6946b9b568a7f5c67fb9a7ef775bbb3.php(46): Twig_Template->displayBlock('blockContent', Array, Array)
#9 \vendor\twig\twig\lib\Twig\Template.php(390): __TwigTemplate_1b4f9619f5c43d58a358dee5589202129a2afea33e85dbf950938f749f8cbf10->doDisplay(Array, Array)
#10 \vendor\craftcms\cms\src\web\twig\Template.php(49): Twig_Template->displayWithErrorHandling(Array, Array)
#11 \vendor\twig\twig\lib\Twig\Template.php(367): craft\web\twig\Template->displayWithErrorHandling(Array, Array)
#12 \vendor\craftcms\cms\src\web\twig\Template.php(31): Twig_Template->display(Array, Array)
#13 \storage\runtime\compiled_templates\da\da8e5f8430fa27f5d100316f9ff89bd64d231217175040d007619d512ef214b1.php(63): craft\web\twig\Template->display(Array, Array)
#14 \vendor\twig\twig\lib\Twig\Template.php(390): __TwigTemplate_b6faf5617e2ccd3b7dfdbc08678649b2eab5a6cc5f94b98453356dccdc895739->doDisplay(Array, Array)
#15 \vendor\craftcms\cms\src\web\twig\Template.php(49): Twig_Template->displayWithErrorHandling(Array, Array)
#16 \vendor\twig\twig\lib\Twig\Template.php(367): craft\web\twig\Template->displayWithErrorHandling(Array, Array)
#17 \vendor\craftcms\cms\src\web\twig\Template.php(31): Twig_Template->display(Array, Array)
#18 \vendor\twig\twig\lib\Twig\Template.php(375): craft\web\twig\Template->display(Array)
#19 \vendor\twig\twig\lib\Twig\Environment.php(289): Twig_Template->render(Array)
#20 \vendor\craftcms\cms\src\web\View.php(331): Twig_Environment->render('feed-me/feeds/_...', Array)
#21 \vendor\craftcms\cms\src\web\View.php(378): craft\web\View->renderTemplate('feed-me/feeds/_...', Array)
#22 \vendor\craftcms\cms\src\web\Controller.php(155): craft\web\View->renderPageTemplate('feed-me/feeds/_...', Array)
#23 \vendor\verbb\feed-me\src\controllers\FeedsController.php(51): craft\web\Controller->renderTemplate('feed-me/feeds/_...', Array)
#24 [internal function]: verbb\feedme\controllers\FeedsController->actionEditFeed('1', NULL)
#25 \vendor\yiisoft\yii2\base\InlineAction.php(57): call_user_func_array(Array, Array)
#26 \vendor\yiisoft\yii2\base\Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#27 \vendor\craftcms\cms\src\web\Controller.php(103): yii\base\Controller->runAction('edit-feed', Array)
#28 \vendor\yiisoft\yii2\base\Module.php(528): craft\web\Controller->runAction('edit-feed', Array)
#29 \vendor\craftcms\cms\src\web\Application.php(282): yii\base\Module->runAction('feed-me/feeds/e...', Array)
#30 \vendor\yiisoft\yii2\web\Application.php(103): craft\web\Application->runAction('feed-me/feeds/e...', Array)
#31 \vendor\craftcms\cms\src\web\Application.php(271): yii\web\Application->handleRequest(Object(craft\web\Request))
#32 \vendor\yiisoft\yii2\base\Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#33 \public_html\index.php(21): yii\base\Application->run()
#34 {main}```

**Comment by noregt** [comment]
Same issue, disabling dev mode is a temporarily solution

**Comment by klick** [comment]
Same here.

**Comment by camelskyscraper** [comment]
Just ran into the same issue trying to demo Feed Me on a new install of Craft Solo 3.0.26.1 in dev mode. Got the error when clicking the +New Feed button, so I can't create a feed. Disabling dev mode solves it.

**Comment by engram-design** [comment]
This should be fixed in [3.0.0-beta.23](https://github.com/verbb/feed-me/releases/tag/3.0.0-beta.23). FYI, this was caused by creating a feed with no sections available.
